.gitignore

```diff
@@ -1 +1 @@
-test.md
+node_modules
```

Comment:
Congratulations on implementing the previous code reviewer suggestion to add `node_modules` to .gitignore. However, you should not remove `test.md`.

- [ ] As it is a project requirement, please make sure to add back `test.md` to .gitignore file

Reply:
Ok, I am going to add it back